projects/vulkan-loader/fuzzers/json_load_fuzzer.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2023 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "cJSON.h"
+#include "loader.h"
+
+/*
+ * Targets the custom version of cJson.
+ */
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  char filename[256];
+  sprintf(filename, "/tmp/libfuzzer.%d", getpid());
+
+  FILE *fp = fopen(filename, "wb");
+  if (!fp) {
+    return 0;
+  }
+  fwrite(data, size, 1, fp);
+  fclose(fp);
+
+  cJSON *json = NULL;
+  loader_get_json(NULL, filename, &json);
+
+  char *json_data = loader_cJSON_Print(json);
```

Comment:
I'd just add this to the existing json fuzzer: https://github.com/google/oss-fuzz/blob/master/projects/vulkan-loader/json_load_fuzzer.c

Reply:
Do we need to check `json != NULL` here?

Reply:
Yes, because it dynamic data, and we need to free it.

Reply:
I mean should we check that the data is non-NULL before passing it on to `loader_cJSON_Print`? What happens if you call `loader_cJSON_Print(NULL)`?

Reply:
Oh, okay I will fix it, because it could lead to false positive NULL dereference. Thank you

Reply:
Fixed

Reply:
Thanks!